tests/plugins/thresholds.yml

```diff
@@ -0,0 +1,18 @@
+# Maybe use - CMD_PLATFORM_SUMMARY = "show platform summary"
+arc-switch1029:
+  cpu_total: 90
```

Comment:
BTW, how a single test can provide its own values? Any performance test/reboot test will utilize CPU at 100% causing such check to fail

Reply:
Fixed. Thresholds can be updated by any test case. For this in test case  user should pass "dut_monitor" fixture.

In test case thresholds can be updated like:
dut_monitor["cpu_total"] = 70
dut_monitor["ram_peak"] = 50
...
Full list of properties are defined in default thresholds file.